Makefile

```
@@ -115,6 +115,6 @@ lint:
 .PHONY: lint-fix
 lint-fix:
 	python3 -c "import black" > /dev/null 2>&1 || python3 -m pip install black
-	python3 -c "import isort" > /dev/null 2>&1 || python3 -m pip install isort
 	python3 -m black --target-version py37 .
-	python3 -m isort .
+	which -s ruff || python3 -m pip install ruff
+	ruff --fix .
```

```suggestion
	python3 -c "import ruff" > /dev/null 2>&1 || python3 -m pip install ruff
	python3 -m ruff --fix .
```
On my machine, when I run `python3 -m pip install ruff`, it isn't automatically available to be run as `ruff`.

This is more flexible.